staging/src/k8s.io/cli-runtime/pkg/resource/builder.go

```diff
@@ -117,20 +115,12 @@ Example resource specifications include:
    '<resource> <name>'
    '<resource>'`)
 
-var LocalResourceError = errors.New(`error: you must specify resources by --filename when --local is set.
+var LocalResourceError = fmt.Errorf(`error: you must specify resources by --filename when --local is set.
```

Comment:
since we don't need to format the string `errors.New` might be better than `fmt.Errorf`

Reply:
The performance of errors.New is better than fmt.Errorf, but don't need to import errors. Actually, I'm not sure which is more suitable.

Reply:
I think `errors.New` is better for this PR since changing it to `fmt.Errorf` will make introduce more changes to `git blame`

Reply:
Thanks for your review, I will update it later.